### PR TITLE
Add secondary info type option to Tile card

### PR DESCRIPTION
**Pull request by MadMaxMcKinney**
## Proposed change

This PR provides an optional setting to the Tile card where a user can hide the state value. This is useful for Entities like Buttons where the state is often not useful information, or for navigational elements. 

This PR by default keeps the state information shown, so existing dashboards should remain unaffected. 

To expand on this feature, and the reason the UX is a dropdown, is I think adding template support or manual overrides for the secondary information in the Tile could be a great next addition! These additional options can be slotted into the dropdown.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

#### Show state
```yaml
type: tile
entity: sensor.sun_next_dawn
secondary_info_type: state
```
ha-tile-second-state

#### Show none
```yaml
type: tile
entity: sensor.sun_next_dawn
secondary_info_type: none
```
ha-tile-second-none

#### UI Configuration
CleanShot 2023-05-29 at 16 59 56@2x

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]
TODO: Branch made [here](https://github.com/MadMaxMcKinney/home-assistant.io/blob/update-tile-docs-secondary-info/source/_dashboards/tile.markdown?plain=1), will make PR once this PR is approved (unless there's a different process here).

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/editor/config-elements/hui-tile-card-editor.ts [hunk 1]
```diff
@@ -256,6 +282,9 @@ export class HuiTileCardEditor
           `ui.panel.lovelace.editor.card.tile.${schema.name}`
         );
 
+      case "secondary_info_type":
+        return "Secondary info type";
```
Comment (spacegaier) [inline src/panels/lovelace/editor/config-elements/hui-tile-card-editor.ts:286]: This should be translated/localized

**Comment by spacegaier** [comment]
I think the YAML parameter should be just `secondary_info` to match what we have in the non tile cards (where we have `secondary_info` and `secondary_info_attribute`).

Also in the non-tiles we have more secondary info options. Shouldn't those then also exist in the title cards?

**Comment by MadMaxMcKinney** [comment]
@spacegaier Good call, I'll add more of the expected secondary options and localize it using the strings from the existing tags.

**Comment by piitaya** [comment]
Hi !
Thanks for the proposal. We discussed that with @bramkragten and we think it can be nice to go a little futher to allow flexibility

Here's the proposal

Allow array of options for `secondary_info` witch can using `state`, `attributes`, `last-changed`, `last-updated`.

```yaml
secondary_info:
   - state
   - preset_mode
   - temperature
```

It will display : `Heat · Eco · 22 °C`

As it's a bigger feature, I will make a PR for that proposal.

**Pull request closed without merging**